.github/workflows/release.yaml

```diff
@@ -0,0 +1,40 @@
+name: Release Workflow
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: 'Version being released'
+        required: true
+      branch:
+        description: 'Branch to release from'
+        required: true
+        default: 'main'
+
+jobs:
+  build:
+    name: Build and push image
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build Image
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: hacbs-test
+        tags: latest ${{ github.event.inputs.release-version }} ${{ github.sha }}
+        containerfiles: |
+          ./Dockerfile
+    - name: Push To quay.io
+      id: push-to-quay
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: ${{ steps.build-image.outputs.image }}
+        tags: ${{ steps.build-image.outputs.tags }}
+        registry: quay.io/redhat-appstudio
+        username: ${{ secrets.HACBS_TEST_QUAY_USER }}
```

Comment:
Just curious, where are the secrets located and who creates them? Sorry I might be missing something

Reply:
The secrets are created in the repository settings, we added them after getting the robot account for the hacbs-test repo.

Reply:
we have a quay robot account added to our repo. its credentials are called here using secrets.HACBS_TEST_QUAY_USER

Reply:
Thanks letting me know!